### ISSUE TITLE
说一下特殊情况下的有趣的用法

### ISSUE THREAD

**Issue by longfei796** [dmMaze/BallonsTranslator#420]
![002](https://github.com/dmMaze/BallonsTranslator/assets/117554192/2d7c023b-2721-4ab5-899d-9130bb985973)
![003](https://github.com/dmMaze/BallonsTranslator/assets/117554192/0dec68bd-3ab4-4c0d-a471-129e11819e0f)
![004](https://github.com/dmMaze/BallonsTranslator/assets/117554192/610ba013-2a68-47b3-8b5b-248113cc9590)
![005](https://github.com/dmMaze/BallonsTranslator/assets/117554192/3b332920-c7f0-4280-9261-4b93f9763532)
![5c](https://github.com/dmMaze/BallonsTranslator/assets/117554192/d63556d9-2287-4691-bd1e-eb537f28e611)
![007](https://github.com/dmMaze/BallonsTranslator/assets/117554192/4ab4da29-830a-42c7-b1e4-213cb680cbfe)
![008](https://github.com/dmMaze/BallonsTranslator/assets/117554192/f2f44702-1811-406b-a4b7-9f469c34e5fe)
![009](https://github.com/dmMaze/BallonsTranslator/assets/117554192/42919899-4942-4d33-8c76-9fee6000d883)
![010](https://github.com/dmMaze/BallonsTranslator/assets/117554192/49e81c45-cc4c-4a3c-ab18-4235d7c20505)
![011](https://github.com/dmMaze/BallonsTranslator/assets/117554192/7e30c10e-d3f5-4cff-b883-8de50102fdb0)
![012](https://github.com/dmMaze/BallonsTranslator/assets/117554192/155cc166-0698-45c4-a00b-9ebcbfabc01d)

**Comment by longfei796** [comment]
不过要注意图片大小规格，不然会一张图大一张图小的情况

**Comment by bropines** [comment]
It is forbidden to display r18 content on github, I advise you to blur the pictures (and also what manga?).

**Comment by bropines** [comment]
For good measure, download a new manga with regular content and use its examples to do everything, we’ve all encountered this here (most of us).

Translator:
为了好的行为，请下载一本带有常规内容的新漫画，并根据其例子做所有事情，我们都遇到了这个情况，并希望提供帮助，但我们有R18内容。这种情况是有的。

**Comment by longfei796** [comment]
> It is forbidden to display r18 content on github, I advise you to blur the pictures (can you say name this manga?).

artist is ahemaru

**Comment by bropines** [comment]
> > It is forbidden to display r18 content on github, I advise you to blur the pictures (can you say name this manga?).
> 
> sorry i do not know that
> 
> artist is ahemaru
> 
> Is that okay now？

Yep. I'm sorry for writing so suddenly. It's just that I was already given a monthly ban on github once, so I'm saving people from this fate)

**Comment by fanglangxinghai** [comment]
大部分的R18的同人CG集，都会有无文字的原图，如果可以将嵌字自动使用在无文字的原图上，那确实会帮助很大。

English translation: Most R18 doujin CG collections come with text-free original images. If it's possible to automatically apply embedded text to these text-free original images, that would indeed be a great help.

**Comment by bropines** [comment]
> @bropines 大部分的R18的同人CG集，都会有无文字的原图，如果可以将嵌字自动使用在无文字的原图上，那确实会帮助很大。
> 
> English translation: Most R18 doujin CG collections have text-free original images. If it's possible to automatically apply embedded text to these text-free original images, that would indeed be a great help.

I've been thinking about it too. as a solution, I had the idea of stupidly copying text blocks with their coordinates when selecting all blocks. Some copy page layout function would be very useful. Either dmMaze will notice our idea or I'll sit down and do something.

**Comment by dmMaze** [comment]
[b8c1474](https://github.com/dmMaze/BallonsTranslator/commit/b8c1474f4e7cc7ef2603f4954ecc9946ab954461) 后 inpainted 目录下可以放命名和尺寸不严格匹配的图了，要保证 inpainted 下的图片排序后顺序和原图是一致的，如果尺寸不对 inpianted 的图会缩放到原图尺寸

**Comment by longfei796** [comment]
> [b8c1474](https://github.com/dmMaze/BallonsTranslator/commit/b8c1474f4e7cc7ef2603f4954ecc9946ab954461) 后 inpainted 目录下可以放命名和尺寸不严格匹配的图了，要保证 inpainted 下的图片排序后顺序和原图是一致的，如果尺寸不对 inpianted 的图会缩放到原图尺寸

您好，作者大佬
inpainted中是修改扣字后的图，如果用作者无背景字的底图替换掉的话，那么对话框不就是还处于未扣字的状态吗
